public/app/features/alerting/unified/mocks/server/handlers/silences.ts

```diff
@@ -4,22 +4,36 @@ import { mockSilences } from 'app/features/alerting/unified/mocks';
 import { MOCK_DATASOURCE_UID_BROKEN_ALERTMANAGER } from 'app/features/alerting/unified/mocks/server/handlers/datasources';
 
 const silencesListHandler = (silences = mockSilences) =>
-  http.get<{ datasourceUid: string }>('/api/alertmanager/:datasourceUid/api/v2/silences', ({ params }) => {
+  http.get<{ datasourceUid: string }>('/api/alertmanager/:datasourceUid/api/v2/silences', ({ params, request }) => {
     if (params.datasourceUid === MOCK_DATASOURCE_UID_BROKEN_ALERTMANAGER) {
       return HttpResponse.json({ traceId: '' }, { status: 502 });
     }
+
+    // Server only responds with ACL if query param is sent
+    const accessControlQueryParam = new URL(request.url).searchParams.get('accessControl');
```

Comment:
Bit of duplication in here and the other handler (and the existing folder one), not sure of the cleanest way to abstract it/whether to bother right now anyway